### DISCUSSION TITLE
note-ml2024fall/svm/

### DISCUSSION THREAD

**Discussion by giscus** [Cgfyufsygsm/cgfyufsygsm.github.io#70] [Announcements]
# note-ml2024fall/svm/

本文主要涉及支持向量机（Support Vector Machine）以及对偶理论。 一键回城。 本文所涉及部分用了 3 个课时来讲述，内容较多。   带约束的优化问题 先介绍一般性的优化问题，进而引出 K.K.T 条件。  等式约束下的优化问题 min⁡xf(x)\min_x f(x)minx​f(x)，s. t. h(x)=0h(x)=0h(x)=0，其中 f,gf,gf,g 都可微。  ∀

https://blog.imyangty.com/note-ml2024fall/svm/

**Comment by Stanleytowne** [comment]
用拉格朗日乘子法求点到平面的距离里面花括号第二个式子应该是 $w^Tx_0 + b = 0$ 吧

**Reply by Cgfyufsygsm** [reply]
打错了，感谢指出

**Reply by Stanleytowne** [reply]
感谢分享！！作者写的太好了

**Comment by pku2213** [comment]
xgg原来这么努力
